src/aggregator/aggregator/follower_flush_mgr.go

```diff
@@ -241,6 +241,20 @@ func (mgr *followerFlushManager) CanLead() bool {
 		mgr.metrics.notCampaigning.Inc(1)
 		return false
 	}
+
+	flushTimes, err := mgr.flushTimesManager.Get()
+	if err != nil {
+		// Error getting flush times, proceed to other logic to determine CanLead.
+		mgr.logger.Warn("Error getting flush times from follower flush manager",
+			zap.Error(err),
+		)
+	} else if flushTimes == nil || len(flushTimes.ByShard) == 0 {
+		// If there are no flush times, the cluster is just coming up (the leader has not
```

Comment:
Sanity: I think this probably makes sense for the new topology case. What about the "nodes just added to the topology case"? Those two nodes will have a new shardset; do we need to ensure that the follower is up at least as long as the leader?

Reply:
Hmm probably cutover/cutoff times handle that, actually; worth reasoning through imo (though it probably just works).